lib/dotcom/system_status.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Dotcom.SystemStatus do
+  @moduledoc """
+  Parent module for the system status feature
+  """
+
+  alias Dotcom.SystemStatus
+
+  @doc """
+  Returns a list of alerts that satisfy the following criteria:
+  - They are for one of the subway or trolley lines (including Mattapan), and
+  - They are either currently active, or will be later today
+  """
+  def subway_alerts_for_today() do
+    subway_alerts_for_today(Timex.now())
+  end
+
+  defp subway_alerts_for_today(now) do
```

Comment:
Maybe rename this function to `subway_alerts_for_day`. That would make more sense with the function above. It's also confusing to name this argument `now` as it can be any time.